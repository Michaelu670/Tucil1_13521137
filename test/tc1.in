A 8 9 Q