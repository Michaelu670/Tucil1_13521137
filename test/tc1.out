42 solutions found
(1 + 3) + (4 * 5)
(1 + 3) + (5 * 4)
((1 + 3) * 5) + 4
(1 + (4 * 5)) + 3
1 + ((4 * 5) + 3)
(1 + (5 * 4)) + 3
1 + ((5 * 4) + 3)
(3 + 1) + (4 * 5)
(3 + 1) + (5 * 4)
((3 + 1) * 5) + 4
3 * ((4 - 1) + 5)
(3 + (4 * 5)) + 1
3 + ((4 * 5) + 1)
3 * ((4 + 5) - 1)
3 * ((5 - 1) + 4)
(3 + (5 * 4)) + 1
3 + ((5 * 4) + 1)
(3 + 5) * (4 - 1)
3 * ((5 + 4) - 1)
4 + ((1 + 3) * 5)
(4 - 1) * (3 + 5)
((4 - 1) + 5) * 3
(4 - (1 - 5)) * 3
(4 - 1) * (5 + 3)
4 + ((3 + 1) * 5)
((4 + 5) - 1) * 3
(4 + (5 - 1)) * 3
((4 * 5) + 1) + 3
(4 * 5) + (1 + 3)
((4 * 5) + 3) + 1
(4 * 5) + (3 + 1)
(5 * (1 + 3)) + 4
((5 - 1) + 4) * 3
(5 - (1 - 4)) * 3
(5 * (3 + 1)) + 4
(5 + 3) * (4 - 1)
((5 + 4) - 1) * 3
(5 + (4 - 1)) * 3
((5 * 4) + 1) + 3
(5 * 4) + (1 + 3)
((5 * 4) + 3) + 1
(5 * 4) + (3 + 1)
Execution time = 1729 microseconds
